waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -530,8 +556,22 @@ proc relayConnectivityLoop*(pm: PeerManager) {.async.} =
     await pm.connectToRelayPeers()
     await sleepAsync(ConnectivityLoopInterval)
 
+proc updateMetrics(pm: PeerManager) {.async.} =
+  # TODO: Hardcoded due to circular dependency
+  let WakuFilterCodec = "/vac/waku/filter/2.0.0-beta1"
```

Comment:
This protocol will very soon be deprecated, so it may be worth rather using the `filter-subscribe` and `filter-push` protocols under filter-v2 instead.

Reply:
resolving as c3d7a3b9584a7d7a64346966fb0e886a8980f18a removed this